src/clinical/clinical-api.ts

```diff
@@ -61,6 +61,20 @@ class ClinicalController {
     res.send(zip.toBuffer());
   }
 
+  @HasProgramReadAccess((req: Request) => req.params.programId)
+  async getProgramClinicalData(req: Request, res: Response) {
+    const programId = req.params.programId;
+    if (!programId) {
+      return ControllerUtils.badRequest(res, 'Invalid programId provided');
```

Comment:
not so much invalid as "missing"

Reply:
this is consistent with the parameter checking of the other methods. comment withdrawn.